std/typecons.d

```diff
@@ -755,6 +755,14 @@ template Tuple(Specs...)
             auto s = a.slice!(1, 3);
             static assert(is(typeof(s) == Tuple!(string, float)));
             assert(s[0] == "abc" && s[1] == 4.5);
+
+            // Phobos issue #15645
```

Comment:
Thanks @tsbockman 

I was not aware. I shall move it into a separate block if there's interest in this approach.